db/schema.sql

```diff
@@ -0,0 +1,31 @@
+DROP DATABASE marzagat;
+CREATE DATABASE marzagat;
+\c marzagat;
+
+CREATE TABLE restaurants (
+  restaurant_id INTEGER,
+  name VARCHAR(50) NOT NULL,
+  neighborhood VARCHAR(50) NOT NULL,
+  google_rating NUMERIC(2,1) NOT NULL,
+  zagat_rating NUMERIC(2,1) NOT NULL,
+  review_count SMALLINT NOT NULL,
+  description TEXT NOT NULL,
+  latitude NUMERIC(7,4) NOT NULL,
+  longitude NUMERIC(7,4) NOT NULL,
+  address VARCHAR(100) NOT NULL,
+  website VARCHAR(50),
+  price_level SMALLINT NOT NULL,
+  type VARCHAR(25) NOT NULL
+);
+
+CREATE TABLE restaurant_photos (
+  photo_id INTEGER,
+  restaurant_id INTEGER NOT NULL,
```

Comment:
It doesn't appear you are using indexes or foreign keys anywhere in your schema. Was curious if that was something you are thinking of adding later for optimization or if you are planning to leave it out?

Reply:
I'll be adding indexes in later, so that I can compare the query times for the table when it is indexed vs when it isn't indexed. The foreign keys is something I need to go back in and add though, thanks for reminding me!